apps/editor/test/unit/ui/modeSwitch.spec.js

```diff
@@ -79,4 +79,18 @@ describe('ModeSwitch', () => {
       expect(modeSwitch._rootElement.style.display).toBe('block');
     });
   });
+
+  describe('form interaction', () => {
+    it('should not trigger form submit on click', () => {
```

Comment:
You don't have to wrap this with `describe`, just write a test suite using `it`.

```js
it('form interaction should not trigger form submit on click', () => {
  // ...
});
```